R/realtime.service.R

```diff
@@ -0,0 +1,182 @@
+                         titleParts$name, ": ", titleParts$advFull)
+    full.title <- title
+    full.text <- paste("This dataset contains a coastal erosion hazards analysis for",
+                        paste0(titleParts$name, "."), "The analysis is based on a storm-impact scaling model",
+                        "that combines observations of beach morphology with hydrodynamic models to predict how sandy beaches,",
+                        "the first line of defense for many coasts exposed to tropical storms and hurricanes, will respond during",
+                        "a direct landfall. Storm-induced total water levels, due to both surge and waves, are compared to beach",
+                        "and dune elevations to determine the probabilities of three types of coastal change - collision (dune erosion),",
+                        "overwash, and inundation.")
+    full.text <- paste(full.text, getSurgeDescription(doc))
+    full.text <- paste(full.text, "Maximum wave heights in 20-m water depth, obtained from the NOAA WaveWatch3",
+                       "model 7-day forecast, were used to compute wave runup elevations at the shoreline.",
+                       "Dune elevations were extracted from lidar topographic surveys.",
+                       "\n\nDisclaimer: This product is based on published research results of the USGS National",
+                       "Assessment of Coastal Change Hazards Project and is intended to indicate the potential",
+                       "for coastal change caused by storm surge and wave runup. This product is based on an",
+                       "analysis that simplifies complex coastal change processes to two important aspects -",
+                       "measured dune elevations and predicted total water levels. As such, the actual changes",
+                       "that occur during extreme storms may be different than what is described here. Results",
+                       "apply to open coast environments and do not consider potential coastal change along",
+                       "inland waters. The public should not base evacuation decisions on this product. Citizens",
+                       "should follow the evacuation advice of local emergency management authorities.")
+    full.publications <- list(data=list(),publications=list(),resources=list())
+    keywords <- ""
+  } else {
+    tiny.text <- paste(titleMap$medium[[subType]], "during", paste0(titleParts$name, ":"), "NHC Adv.", titleParts$advNum)
+    medium.title <- paste(titleMap$medium[[subType]])
+    
+    if (subType == "pcol" | subType == "povw" | subType == "pind") {
+      description <- getPCOIDescription(subType, titleParts, doc)
+      full.title <- paste(titleMap$medium[[subType]], "during", paste0(titleParts$name, ":"), titleParts$advFull)
+      medium.text <- description$medium
+      full.text <- description$full
+    } else if (subType == "dhigh" | subType == "dlow") {
```

Comment:
for DLOW, tiny text should be "Dune toe elevation prior to ***Storm Name***" rather than follow pattern in line 59. (actually seems to be redundant with the full title)